docs/userguide/locale/index.md

```diff
@@ -97,17 +97,20 @@ The optional four-letter script code follows the language code. If specified, it
 should be a valid script code as listed on the 
 [Unicode ISO 15924 Registry](https://www.unicode.org/iso15924/iso15924-codes.html).
 
-### Country code
+### Region code
 
 There are often different language conventions within the same language. For
 example, Spanish is spoken in many countries in Central and South America but
 the currencies are different in each country. To allow for these differences
-among specific geographical, political, or cultural regions, locales are
-specified by two-letter, uppercase codes. For example, "ES" represents Spain and
-"MX" represents Mexico. The two letter country code uses the
-[ISO-3166](https://www.iso.org/iso-3166-country-codes.html) standard.
-
-Java supports two letter country codes that uses ISO-3166 and UN M.49 code.
+among specific geographical, political, or cultural regions, locales follow
+the [BCP 47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt) convention of specifying
```

Comment:
```suggestion
the [BCP 47](https://www.rfc-editor.org/info/bcp47) convention of specifying
```
--> nicer page, and easy access to the HTML versions

Reply:
done